src/lib/shell/commands/Ota.cpp

```diff
@@ -64,12 +52,8 @@ CHIP_ERROR NotifyImageHandler(int argc, char ** argv)
     VerifyOrReturnError(GetRequestorInstance() != nullptr, CHIP_ERROR_INCORRECT_STATE);
     VerifyOrReturnError(argc == 4, CHIP_ERROR_INVALID_ARGUMENT);
 
-    const FabricIndex fabricIndex       = static_cast<FabricIndex>(strtoul(argv[0], nullptr, 10));
```

Comment:
@Damian-Nordic I think this will break Notify since persistence isn't supported yet. I think between you or @hrajendran-samsung, persistence support will be added soon. Ok to remove this for now?